examples/plot_anomaly_comparison.py

```diff
@@ -0,0 +1,120 @@
+"""
+========================================================================
+Comparing different anomaly/novelty detection algorithms on toy datasets
+========================================================================
+
+This example shows characteristics of different
+anomaly/novelty detection algorithms on 2D datasets. Datasets contain
+one or two modes (regions of high density) to illustrate the ability
+of algorithms to cope with multimodal data.
+
```

Comment:
I think you should state:

15% of samples in each dataset are generated as random uniform noise. We display decision boundaries with a threshold that would mark 15% of training samples as outliers.

Although admittedly I'm a bit confused by this threshold...